src/App.tsx

```diff
@@ -15,36 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum ESortType {
+  ALPHABETICALLY = 'alphabetically',
+  LENGTH = 'by length',
+  DEFAULT = '',
+}
+
+interface IOptions {
+  sortBy: ESortType;
+  isReversed: boolean;
+}
```

Comment:
what `I/E` mean?

Reply:
I - interface, E - enum. It is useful for importing such types. You can understand what type it is

Reply:
I never faced this indication before tbh and I don't think this is necessary.

I mean if we have variable typed with SortType it is kind of clear that we're dealing with enum by its name. Also the thing with typescript is that it displays errors instantly if you used some type incorrectly. 

So I'd say that it is better to keep names clean because it is almost the same as naming things like `goodsArr` or `shouldRenderListBool`